tdp_vars_defaults/livy-spark3/livy-spark3.yml

```diff
@@ -75,6 +72,21 @@ livy_env:
   LIVY_LOG_DIR: "{{ livy_log_dir }}"
   LIVY_PID_DIR: "{{ livy_pid_dir }}"
 
+# Livy logging configuration
+# Root logger level should be: [FATAL| ERROR| WARN| INFO| DEBUG| TRACE]
+livy_log_dir: /var/log/livy-spark3
```

Comment:
We put this kind of directory inside `tdp-custer` variables in `tdp-collection` no ?

Reply:
`tdp-cluster` in `tdp-collections-extras` has lots of stuff missing for the moment. All ports have to be moved there too.

Reply:
ok no problem if we move it later.